lib/flipper/ui/views/feature.erb

```diff
@@ -9,7 +9,7 @@
     <div class="col">
       <div class="card">
         <div class="card-body">
-          <%= @feature.description %>
+          <%== Sanitize.fragment(@feature.description, Sanitize::Config::BASIC) %>
```

Comment:
Seems like [Sanitize::Config::BASIC](https://github.com/rgrove/sanitize#sanitizeconfigbasic) is sensible. Can potentially make this an options, but this seems good enough for now.